.github/workflows/build-and-push-images.yaml

```diff
@@ -23,10 +23,12 @@ jobs:
           - component-name: dataset-initializer
             dockerfile: cmd/initializers/dataset/Dockerfile
             platforms: linux/amd64,linux/arm64
+          - component-name: deepspeed-runtime
```

Comment:
The image build takes ~ 40 minutes due to image size.
Is there any way for us to speed it up ?